solidity/test/LiquidityProtectionTargetAmount.js

```diff
@@ -0,0 +1,181 @@
+const { expect } = require('chai');
```

Comment:
Can you move this to one of the existing test files? There are already so many LP test files and it makes it harder to test

Reply:
Couldn't find anyone really suitable.

Reply:
In addition to that, this test relies on a very simple one-time initialization step, in order to speed up testing of the actual feature at stake, whereas all other test files of this contract already utilize more complex initialization which would be redundant in this case.
Finally, I don't see any real benefit in having fewer files, where each file is larger. If anything, it makes it harder to maintain them (just like you would try to avoid writing too long functions, classes, etc).
Can you please explain the benefit of pouring the contents of this file into another file in this case?

Reply:
You can use scopes to have dozens of different initialization logic in the same file. The rationale behind keeping the same component tests in the same file (other than being the de-facto standard) is that it allows much easier management (e.g., running all X tests at the same time, configuring watchers that if Y changes - run X again, etc.). In addition to that, the same component tests can usually reuse similar code (which is heavily duplicated today).

Reply:
@barakman move to one of the other files, too many of them, it's becoming unmanageable to run all and only liquidity protection tests

Reply:
Done.